Dockerfile

```diff
@@ -19,7 +19,7 @@ ARG GROUPID=1000
 ARG USERNAME=aca
 ARG USERID=33009
 
-COPY ./docker/nginx.conf /etc/nginx/nginx.conf
+COPY ./docker/nginx.conf /
```

Comment:
the path of the config is wrong, it should be placed to the /etc/nginx folder inside the image

Reply:
the nginx folder is owned by root. I will try if I can add the alfresco user to the nginx folder